README.md

```diff
@@ -92,3 +92,68 @@ Options:
+ staking operations etc.
+
+
+If however, you are more interested in intereacting with the contracts in the Moonbeam emulated EVM environment, but not so much parachain staking or XCM, you can run
+the forked network as a solo node in dev mode. This allows for manual sealing of blocks which dramatically reduces execution times of tests (reduction of 12s blocktime into milliseconds).
+
+```
+npm run fork-solo
+```
+
+### Further Examples
+
+Calling the script directly can be done via: `npx ts-node ./src/tools/run-moonbeam-fork.ts`
+
+The minimal command is: `npx ts-node ./src/tools/run-moonbeam-fork.ts -n moonbeam`
+
+By default the polkadot and moonbeam binaries will be downloaded from github if none found in the `binaries` folder, but if for whatever reason you need to provide your own (e.g. you are on an Apple Silicon chip)
+use the `--polkadot-binary` option to provide the path to the binary to use (or copy them into the folder). If doing this option, make sure the correct binary version is supplied via `--moonbeam-version` and 
+`--polkadot-version` respectively, otherwise the script will still attempt to download the latest.
+
+
+The full list of options can be found in the `--help` :
+```
+      --help                      Show help                            [boolean]
+      --version                   Show version number                  [boolean]
+  -n, --network                   Network to retrieve the exported state for
+                                                             [string] [required]
+      --latest                    Will verify if a latest snapshot is available
+                                  and download it     [boolean] [default: false]
+      --purge                     Will delete previous execution database
+                                                      [boolean] [default: false]
+      --solo                      Will run the network as a single manual-sealed
+                                  dev node            [boolean] [default: false]
+      --ephemeral                 Will leave the network running after setup has
+                                  completed           [boolean] [default: false]
+      --purge-specs               Will delete previous generated specs
+                                                      [boolean] [default: false]
+  -m, --moonbeam-binary           Binary file path or of the moonbeam node
+                                       [string] [default: "./binaries/moonbeam"]
+  -p, --polkadot-binary           Binary file path of the polkadot node
+                                       [string] [default: "./binaries/polkadot"]
+      --polkadot-version, --pver  Client version number for Polkadot binary
+                                                    [string] [default: "latest"]
+      --moonbeam-version, --mver  Client version number for Moonbeam binary
```

Comment:
I thought this could be combined with the --moonbeam-binary to support both case:

--moonbeam-binary => (if filepath => use the file; otherwise try to download the given version)

Reply:
Done